From 4fd67b6adb7c1d8524ba17d1a0b3894f901555a9 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Thu, 13 May 2021 15:23:16 +0800
Subject: [PATCH] Makefile.am: only build dhcrelay

Drop client and server build as we don't need them.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index ed692a5..34f9772 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -38,7 +38,7 @@ endif
 
 # Use an autoconf substitution vs an automake conditional here
 # to fool automake when the bind directory does not exist.
-SUBDIRS = @BINDSUBDIR@ includes tests common omapip client dhcpctl relay server
+SUBDIRS = @BINDSUBDIR@ includes common omapip relay
 
 DIST_SUBDIRS = $(SUBDIRS) keama
 
-- 
2.25.1

